pkg/controller/builder/pod_builder.go

```diff
@@ -10,13 +10,13 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
-func BuildActiveGatePodSpecs(instance *v1alpha1.ActiveGate, tenantInfo *dtclient.TenantInfo, kubeSystemUID types.UID) (corev1.PodSpec, error) {
-	sa := MonitoringServiceAccount
+func BuildActiveGatePodSpecs(instance *v1alpha1.DynaKube, tenantInfo *dtclient.TenantInfo, kubeSystemUID types.UID) (corev1.PodSpec, error) {
+	serviceAccount := MonitoringServiceAccount
```

Comment:
There is a pseudo-convention in Go to use shorter names for local variables if they don't really matter. To be fair the guideline is quite arbitrary, but I agree that it makes things easier to read, since there is less noise.

Global variables can have full names.